rest_api/openshift_v1.adoc

```diff
@@ -9278,7 +9278,7 @@ Patch is provided to give a concrete name and type to the Kubernetes PATCH reque
 |certificate|provides certificate contents|false|string|
 |key|provides key file contents|false|string|
 |caCertificate|provides the cert authority certificate contents|false|string|
-|destinationCACertificate|provides the contents of the ca certificate of the final destination.  When using re-encrypt termination this file should be provided in order to have routers use it for health checks on the secure connection|false|string|
+|destinationCACertificate|provides the contents of the ca certificate of the final destination.  When using re-encrypt termination this file should be provided in order to have routers use it for health checks on the secure connection|true|string|
```

Comment:
@jkaurredhat The description here "When using re-encrypt termination this file should be provided [...]" makes it sound like it's more accurate to be listed as false overall, and only required when using re-encryption. But that's just my interpretation; Engineering likely would have to weigh in on what's more valid for "Required" in that context.

@ahardin-rh BTW, the content for the API Guide is generated from the origin product itself (see https://github.com/openshift/origin/tree/master/api), which is run periodically (though not as frequent as we probably should) to update this doc. So, any manual changes made directly here would be overwritten next time we generated the doc. If we do change this value here, we would want to follow up and see about getting it changed in the origin product itself so that it persists next time we re-generate.